introduction.md

```diff
@@ -0,0 +1 @@
+hi my name is bailey. I'm from the cayman islands.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
updated intro